applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt

```diff
@@ -11,15 +11,15 @@ This USB interface has the following functions:
 * CMSIS-DAP 2.1 compliant debug probe interface for accessing the nRF91 SiP
 
 COM Ports
-====================
+=========
 
 This USB interface exposes two COM ports mapped to the physical UART interfaces between the nRF91 Series and nRF52840 devices.
-When opening these ports manually (without using the LTE Link Monitor), be aware that the USB COM port baud rate selection is applied to the UART.
+When opening these ports manually (without using the nRF Connect Serial Terminal), be aware that the USB COM port baud rate selection is applied to the UART.
 
 Bluetooth® LE Central UART Service
 ==================================
 
-This device advertises as "Thingy:91 UART".
+This device advertises as configured in Config.txt BLE_NAME, default "Thingy:91 UART".
```

Comment:
```suggestion
This device advertises as configured in ``BLE_NAME`` in :file:`Config.txt`, default being "Thingy:91 UART".
```

Reply:
@umapraseeda This text is used in the `README.txt` file on the USB drive for the device and I don't think we can use markup here. I can change the text to "This device advertises as configured in BLE_NAME in Config.txt, default being "Thingy:91 UART"."